aws/EC2/ec2-how-to/index.rst

```diff
@@ -0,0 +1,7 @@
+How-to guides
```

Comment:
nits:
* could we use lower-case naming in file path?
* why `ec2-how-to`? it's already in the `EC2` folder so `how-to` would be good enough

Reply:
* That's a good catch, all file paths are now lower-case.
*  Just using `how-to` leads to sphinx warnings since multiple locations have `how-to`. So I have left them as is.